### ISSUE TITLE
another deprecation problem with matplotlib

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#590]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/590) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

```
Fatal error in MSS 1.9.5. on Linux-4.19.0-10-amd64-x86_64-with-glibc2.10
Python 3.8.6 | packaged by conda-forge | (default, Oct  7 2020, 19:08:05) 
[GCC 7.5.0]

Please report bugs in MSS to https://bitbucket.org/wxmetvis/mss

Information about the fatal error:

Traceback (most recent call last):
  File "mss_pyui.py", line 434, in create_new_view
    view_window = topview.MSSTopViewWindow(model=self.active_flight_track)
  File "/home/icg173/src/mss/mslib/msui/topview.py", line 168, in __init__
    self.setupUi(self)
  File "/home/icg173/src/mss/mslib/msui/qt5/ui_topview_window.py", line 21, in setupUi
    self.mpl = MplTopViewWidget(self.centralwidget)
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 1205, in __init__
    super(MplTopViewWidget, self).__init__(
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 492, in __init__
    self.navbar = NavigationToolbar(self.canvas, self, sideview)
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 243, in __init__
    super(NavigationToolbar, self).__init__(canvas, parent, coordinates)
  File "/home/icg173/anaconda3/lib/python3.8/site-packages/matplotlib/backends/backend_qt5.py", line 687, in __init__
    NavigationToolbar2.__init__(self, canvas)
  File "/home/icg173/anaconda3/lib/python3.8/site-packages/matplotlib/backend_bases.py", line 2857, in __init__
    init()
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 417, in _init_toolbar
    self.basedir = os.path.join(matplotlib.rcParams['datapath'], 'images')
  File "/home/icg173/anaconda3/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py", line 203, in __set__
    return super().__set__(instance, value)
AttributeError: can't set attribute
```

happens with matplotlib version 3.3.2

‌

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

This probably requires quite a bit of a “clean” redesign of our toolbar hacking.

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Occurs for all matplotlib version > 3.3.0. Some other problems cropped up for the old code and matplotlib 3.2.x, but that should only be warnings…  
  
Pinning matplotlib to >3.3.0 ensure that the new code works upon a new install/update.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Rewrote toolbar to accommodate matplotlib 3.3.x

Fix issue #590
